arelle/PluginManager.py

```diff
@@ -493,7 +494,15 @@ def pluginClassMethods(className: str) -> Iterator[Callable[..., Any]]:
             yield method
 
 def addPluginModule(url):
-    moduleInfo = moduleModuleInfo(url)
+    moduleInfo = None
+    unloaded_arelle_plugins = entry_points(group='arelle.plugin', name=url)
+    if unloaded_arelle_plugins:
+        if len(unloaded_arelle_plugins) != 1:
+            raise Exception(f'multiple pip installed plugins with name {url} in group arelle.plugin')
```

Comment:
Looking at the code path there doesn't appear to be a try except block around this. Can you look at logging an error instead? We typically try to fail gracefully when plugins can't be loaded.